OKR.md

```diff
@@ -0,0 +1,39 @@
+# Nodejs.dev Objectives and Key Results
+
+The Nodejs.dev project uses Objectives and Key Results to try to provide direction and guidance to the work we to, enabling our own work and the work of our initiatives to have a north star for us to ensure we're working towards.
+
+## Key Concepts
+
+OKRs consist of two elements:
+
+- Objectives
+  - Objectives are qualatative, ambitious, and memorable definitions of the things we as a committee would like to achieve.
+- Key Results
+  - Key Results are a set of metrics by which we can measure progress toward their objectives. Like objectives, they should be ambitious and overestimate what we can accomplish.
+
+### External References
+
+Here are some external references that may help provide further context on OKRs.
+
+- [Guide: Set goals with OKRs](https://rework.withgoogle.com/guides/set-goals-with-okrs/steps/introduction/)
+- [What is OKR?](https://felipecastro.com/en/okr/what-is-okr/)
+- [What Are OKRs? How to Use OKRs for Planning (and Achieving) Your Goals](https://plan.io/blog/what-are-okrs/)
+
+## OKRs (2020 Set 1):
+
+- **Objective:** Continuity of process
+
+  - **Key Result:** Draft and maintain technical administration summary and how-to guides covering staging and CI.
+  - **Key Result:** Consider using issue templates to facilitate operations.
+
+- **Objective:** Identify areas of progress and future goals
+
+  - **Key Result:** Create a feature roadmap
+  - **Key Result:** Track features & priorities in one place (project?)
+  - **Key Result:** Make available the top two features in nodejs.org:
+    1. Docs — https://nodejs.org/en/docs/
+    2. Download — https://nodejs.org/en/download/
+
+- **Objective:** Make explicit the channels of interfacing internally and with larger org
+  - **Key Result:** Write down implicit team norms
+  - **Key Result:** How do we propose and review non-feature changes in PRs?
```

Comment:
I seem to misunderstand the discussion. Is this about using PRs to propose changes instead of Issues? If so, what friction are we looking to solve?